src/NHibernate.Test/Async/NHSpecificTest/GH1180/FixtureByCode.cs

```diff
@@ -16,7 +16,7 @@
 namespace NHibernate.Test.NHSpecificTest.GH1180
 {
 	using System.Threading.Tasks;
-	[KnownBug("NH-3847 (GH-1180)")]
+	//NH-3847 (GH-1180)
```

Comment:
Dont need this commet. It is alterady in that namespace

Reply:
It's been added to mention NH-3847... Ok GH part deleted